packages/table/preview/index.tsx

```diff
@@ -683,6 +767,30 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
         this.store.setColumnName(columnIndex, value);
     }
 
+    private handleScrollToButtonClick = () => {
+        const { scrollToRowIndex, scrollToColumnIndex, scrollToRegionType } = this.state;
+
+        let region: IRegion;
+        switch (scrollToRegionType) {
+            case RegionCardinality.CELLS:
+                region = Regions.cell(scrollToRowIndex, scrollToColumnIndex);
+                break;
+            case RegionCardinality.FULL_ROWS:
+                region = Regions.row(scrollToRowIndex);
+                break;
+            case RegionCardinality.FULL_COLUMNS:
+                region = Regions.column(scrollToColumnIndex);
+                break;
+            case RegionCardinality.FULL_TABLE:
+                region = Regions.table();
+                break;
+            default:
+                return;
+        }
+
+        this.tableInstance.scrollToRegion(region);
```

Comment:
is invoking methods on the table ref the standard way for actions like this? do we have other components/features that do this? is this what users will expect?

Reply:
i guess the resizeRows does this. still, is this the idiomatic blueprint way to invoke actions on react components?

Reply:
Yeah, there are a few instance methods around the codebase, each of which operates on a `ref`. The goal here was to provide a moment to briefly control the scroll without forcing the caller to control it forever and always (would be annoying and unnecessary). See:

```tsx
// Table
// =====

resizeRowsByTallestCell(columnIndices?: number | number[]): void;

// Toast
// =====

clear(): void;
dismiss(key: string): void;
getToasts(): IToastProps[];
show(props: IToastProps): string;
update(key: string, props: IToastProps): void;

// Tree
// ====

getNodeContentElement(nodeId: string | number): HTMLElement | undefined;
```